launcher/ui/pages/global/MinecraftPage.cpp

```diff
@@ -109,6 +114,16 @@ void MinecraftPage::loadSettings()
     ui->useNativeOpenALCheck->setChecked(s->get("UseNativeOpenAL").toBool());
     ui->useNativeGLFWCheck->setChecked(s->get("UseNativeGLFW").toBool());
 
+    ui->enableFeralGamemodeCheck->setChecked(s->get("EnableFeralGamemode").toBool());
+    ui->enableMangoHud->setChecked(s->get("EnableMangoHud").toBool());
+    ui->useDiscreteGpuCheck->setChecked(s->get("UseDiscreteGpu").toBool());
+
+    #if !defined(Q_OS_LINUX)
+    ui->enableFeralGamemodeCheck->setDisabled(true);
+    ui->enableMangoHud->setDisabled(true);
+    ui->useDiscreteGpuCheck->setDisabled(true);
+    #endif
```

Comment:
Maybe hide these instead. Disabled options should only communicate that something is not available *currently*. But these will be permanently disabled on other platforms, so they just shouldn't be there at all.

```suggestion
#if !defined(Q_OS_LINUX)
    ui->enableFeralGamemodeCheck->setVisible(false);
    ui->enableMangoHud->setVisible(false);
    ui->useDiscreteGpuCheck->setVisible(false);
#endif
```